### PR TITLE
Remove Builder from ListItem

### PR DESCRIPTION
**Pull request by Hofer-Julian**
It does not make sense there

### CODE REVIEW

File: gtk4/Gir.toml [hunk 1]
```diff
@@ -2528,3 +2527,10 @@ trait_name = "GtkWindowExt"
     [[object.signal]]
     name = "close-request"
     inhibit = true
+
+[[object]]
```
Comment (bilelmoussaoui) [inline gtk4/Gir.toml:2531]: The items must be ordered otherwise the CI won't be happy

**Review by bilelmoussaoui — CHANGES_REQUESTED** [review]

**Comment by bilelmoussaoui** [comment]
do you mind disabling the builder for BuilderListItemFactory as well?

**Comment by bilelmoussaoui** [comment]
You need to update your gir submodule. The versions files changes shouldn't occur as gir was updated today.

**Comment by bilelmoussaoui** [comment]
I have replaced it with #457

**Comment by Hofer-Julian** [comment]
Thanks

**Pull request closed without merging**